Sources/SwiftKotlinFramework/KotlinTokenizer.swift

```diff
@@ -1078,3 +1084,32 @@ public struct InvertedCondition: ASTTokenizable {
     public let condition: Condition
 }
 
+// function used to remove default arguments from override functions, since kotlin doesn't have them
+private func removeDefaultArgsFromParameters(tokens:[Token]) -> [Token] {
+    var newTokens = [Token]()
```

Comment:
I am moving this function to a private method inside the main class. There is no real benefit of it since we are not inheriting for now, but just for consistency with the rest.